db/seeds.rb

```diff
@@ -5,3 +5,22 @@
+
+Sed in elementum libero. Maecenas eleifend justo eget mauris ultricies, in posuere orci bibendum. Cras vel justo metus. Vivamus non fringilla erat. Nullam auctor sit amet neque sit amet auctor. Vivamus eget nisl a massa viverra convallis ac quis libero. Curabitur hendrerit, velit eget efficitur fermentum, mi eros interdum nisl, ut lacinia urna ipsum in lectus. Integer sollicitudin, lectus eget convallis finibus, elit risus blandit tellus, eget euismod turpis velit ac nibh. Integer scelerisque dui nec justo auctor, et egestas est scelerisque. Aenean in dolor at arcu venenatis finibus.
+
+Sed tincidunt, arcu vel sollicitudin semper, purus nunc bibendum neque, vel consectetur nisi elit et libero. Sed mollis egestas turpis, eu varius orci fermentum eget. Aliquam quis arcu at nisi venenatis feugiat. Sed in lectus ac lectus scelerisque malesuada. Vivamus semper')
+
+post_one = Post.create(user: user_one, title: 'Lorem Ipsum', text: 'Nam at justo hendrerit, vehicula metus non, facilisis dui. Aenean vitae metus ligula. Cras dignissim congue ligula, non bibendum nisi scelerisque nec. Aliquam erat volutpat. Fusce euismod ex vitae augue luctus, nec accumsan nisi consequat. Ut sit amet urna aliquam, volutpat mi nec, varius libero. Fusce eu lorem ut enim maximus tincidunt. Cras consequat, sapien ac sollicitudin posuere, felis nulla sollicitudin ipsum, sit amet feugiat elit felis et massa. Suspendisse tristique nulla id diam dictum hendrerit. Vivamus scelerisque ipsum vitae purus euismod venenatis. Aenean sodales a dui ut viverra. Nullam malesuada velit non libero mattis feugiat. Nullam dapibus purus nec nisi volutpat, nec facilisis purus congue. Etiam vel quam vel odio pharetra feugiat at eu purus. Phasellus maximus elit vel ligula dictum, vitae suscipit velit bibendum.
+
+Sed in elementum libero. Maecenas eleifend justo eget mauris ultricies, in posuere orci bibendum. Cras vel justo metus. Vivamus non fringilla erat. Nullam auctor sit amet neque sit amet auctor. Vivamus eget nisl a massa viverra convallis ac quis libero. Curabitur hendrerit, velit eget efficitur fermentum, mi eros interdum nisl, ut lacinia urna ipsum in lectus. Integer sollicitudin, lectus eget convallis finibus, elit risus blandit tellus, eget euismod turpis velit ac nibh. Integer scelerisque dui nec justo auctor, et egestas est scelerisque. Aenean in dolor at arcu venenatis finibus.
+
+Sed tincidunt, arcu vel sollicitudin semper, purus nunc bibendum neque, vel consectetur nisi elit et libero. Sed mollis egestas turpis, eu varius orci fermentum eget. Aliquam quis arcu at nisi venenatis feugiat. Sed in lectus ac lectus scelerisque malesuada. Vivamus semper')
+
+6.times do
+    Comment.create(user_id: user_one.id, post_id: post_one.id, text: 'cuowevg owvwuei huiobqugog ey cfgiugey efuuioqcnfu qrb ryqfo hpe...')
+    Comment.create(user_id: user_two.id, post_id: post_two.id, text: 'cuowevg owvwuei huiobqugog ey cfgiugey efuuioqcnfu qrb ryqfo hpe...')
+end
```

Comment:
## [OPTIONAL]
- [ ] As I have said In the comment above kindly revise your association, here in the seed file you are using user_id to associate the post to the User but in your database, the association of the User and the Post is done using the author_id foreigner_key. Kindly revise this and do the same for the Comment

![image](https://github.com/BANCUNGUYE66/Blog-app/assets/79058364/b2d2da96-2dcb-4124-a571-f203338cc624)